fv3core/stencils/d_sw.py

```diff
@@ -85,24 +103,72 @@ def flux_capacitor(
         yflux = yflux + fy
 
 
+@gtscript.function
+def corner_ke(
```

Comment:
Oh I think I see what you're saying, it's possible this logic is used in multiple places but just not right now because of an unported option. I'll investigate the Fortran code and see if that's the case, if there are I'll return it back or place a comment here with a reference to the unported options.